backend/pong/match/tests/test_message_exceptions.py

```
@@ -0,0 +1,82 @@
+        # receive_jsonメソッドを実行
+        await consumer.receive_json(message)
+
+        # logger.warningが呼ばれたかをチェック
+        mock_logger.warning.assert_called_with(
+            'Missing required key: \'category\', message: {}'
+        )
+        
+    @patch('backend.pong.match.consumers.logger')  # loggerをモック
+    @patch('backend.pong.match.consumers.MatchHandler')  # MatchHandlerをモック
+    @patch('channels.layers.get_channel_layer')  # チャネルレイヤーをモック
+    async def test_receive_json_invalid_data(self, mock_get_channel_layer, MockMatchHandler, mock_logger):
```

Ruff (0.8.2)

Argument name `MockMatchHandler` should be lowercase

(N803)